cuda-compatible-runtime.spec

```diff
@@ -0,0 +1,26 @@
+### RPM external cuda-compatible-runtime 1.0
+
+%define branch master
+%define commit 18d5a51bfb32fbb7b765dd2eecd14e193cce8126
+
+Source: git+https://:@gitlab.cern.ch:8443/cms-patatrack/%{n}.git?obj=%{branch}/%{commit}&export=%{n}&filter=test.c&output=/%{n}-%{realversion}.tgz
```

Comment:
this still means that only users with read access to this repo can build this external ... right ? This will stop other users to build externals

Reply:
please change the `filter=test.c` to `filter=./test.c` . cmsBuild uses https://github.com/cms-sw/pkgtools/blob/V00-32-XX/cmsBuild#L531 to filter the files which means they will have `./` infront